tor/core/inbox.py

```diff
@@ -57,15 +57,14 @@ def check_inbox(config):
         if 'reload' in item.subject.lower():
             item.mark_read()
             reload_config(item, config)
-            item.reply(
-                'Config reloaded!'
-            )
```

Comment:
Yeah, we probably should be replying that it was reloaded if it wasn't (e.g., non-authorized user DM's the command)